crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -176,6 +176,49 @@ def create_proposal
     base
   end
 
+  def prepare_nodes_for_os_upgrade
+    nodes_to_upgrade = []
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `nodes_to_upgrade`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)